lib/engines/content_block_manager/app/models/content_block_manager/content_block/edition.rb

```diff
@@ -31,6 +31,10 @@ def render
         ).render
       end
 
+      def first_class_details
+        details.select { |_k, v| v.is_a?(String) }
```

Comment:
🤔  Is it always going to be the case that entries with String values are "first class" / important enough to be shown on the summary card, and entries with other values are not?

Or is this method likely to develop special cases?

Reply:
For now, almost definitely. We can revisit as we develop new schemas